src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -1047,4 +1040,49 @@ internal override void ForceComplete(SourceLocation locationOpt, CancellationTok
             var unused = this.ExplicitDefaultConstantValue;
         }
     }
+
+    internal sealed class SourceComplexParameterSymbolWithCustomModifiers : SourceComplexParameterSymbol
```

Comment:
I didn't think we could define parameters with customer modifiers in source.  Can you give an example of when this type would be used?